src/Umbraco.Infrastructure/Trees/TreeNode.cs

```diff
@@ -113,7 +114,11 @@ public string IconFilePath
                     return Current.IOHelper.ResolveUrl("~" + Icon.TrimStart('~'));
 
                 //legacy icon path
-                var backOfficePath = Current.Configs.Global().GetUmbracoMvcArea(Current.HostingEnvironment);
+
+                // TODO: replace this when we have something other than Current.Configs available
```

Comment:
This `TODO` is also no longer required.

Reply:
These TODOs are still required, as I just replaced `Current.Configs` with their specific config class on `Current`